docs/gpa.md

```diff
@@ -568,7 +569,7 @@ This requires the [Participation Fees](https://github.com/open-contracting-exten
   </table>
 </div>
 
-## [Article XIV](https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm#articleXIV) Electronic Auctions
+  ## [Article XIV](https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm#articleXIV) Electronic Auctions
```

Comment:
Why add whitespace here? I think it breaks the Markdown.

Reply:
Fixed.